controllers/componentversion_controller.go

```diff
@@ -312,11 +322,11 @@ func (r *ComponentVersionReconciler) checkVersion(ctx context.Context, octx ocm.
 	if err != nil {
 		return false, "", fmt.Errorf("failed to parse reconciled version: %w", err)
 	}
-	log.V(4).Info("current reconciled version is", "reconciled", current.String())
+	logger.V(4).Info("current reconciled version is", "reconciled", current.String())
```

Comment:
can make those debug level logs?

Reply:
They are. They are `V(4)`. Or what else do you mean?